src/frob.cc

```diff
@@ -90,6 +89,96 @@ unsigned long ByteData(unsigned long addr) {
 static_assert(false, "uh oh, bad PYFLAME_PY_VERSION");
 #endif
 
+#if PYFLAME_PY_VERSION >= 34
+std::string StringDataPython3(pid_t pid, unsigned long addr) {
+  // TODO: This function only works for Python >= 3.3. Is it also possible to
+  // support older versions of Python 3?
+
+  // TODO: Can we guarantee that the same padding is used for the bitfield?
```

Comment:
It *should* work if Python and Pyflame are compiled using the same compiler .This is a good point though.